apps/schools/domains/common/components/Input/index.tsx

```diff
@@ -58,7 +59,20 @@ export const Input: React.FC<CustomInputProps> = (props) => {
                 />
             </div>
         )
-    } else {
+    } else if (customType === 'inputPassword') {
+        return (
+            <div className={defaultStyles.inputContainer}>
+                <label>{label}</label>
+                <BaseInput.Password
+                    iconRender={(visible: boolean) => (visible ? <EyeTwoTone /> : <EyeInvisibleOutlined />)}
+                    className={defaultStyles.input}
+                    {...restProps}
+                    placeholder={placeholder}
+                    data-testid="input"
```

Comment:
why do we need data-testid prop? 
value of this prop should be password-input
I would delete this string and pass with restProps